evap/staff/urls.py

```diff
@@ -23,12 +23,14 @@
     url(r"^semester/(\d+)/course/create$", course_create),
     url(r"^semester/(\d+)/course/(\d+)/edit$", course_edit),
     url(r"^semester/(\d+)/course/(\d+)/delete$", course_delete),
-    url(r"^semester/(\d+)/course/(\d+)/review$", course_review),
-    url(r"^semester/(\d+)/course/(\d+)/review/(\d+)$", course_review),
     url(r"^semester/(\d+)/course/(\d+)/email$", course_email),
     url(r"^semester/(\d+)/course/(\d+)/unpublish$", course_unpublish),
     url(r"^semester/(\d+)/course/(\d+)/preview$", course_preview),
     url(r"^semester/(\d+)/course/(\d+)/comments$", course_comments),
+    url(r"^semester/(\d+)/course/(\d+)/comments/(\w+)$", course_comments),
```

Comment:
can't you use these ?foo=bar parameters? this looks like a new page, but it's not.